tests/cases/example.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE demo (name string TAG, value double NOT NULL, t timestamp NOT NULL, TIMESTAMP KEY(t)) ENGINE=Analytic with (enable_ttl='false');
```

Comment:
If I have a new test case, whether should I create a new file or modify existing one?

Is there any suggested guideline on how to group test cases?

Reply:
I'll write a readme about how to use

Reply:
Added